TeachEasy/src/teacheasy/data/ButtonData.java

```diff
@@ -0,0 +1,85 @@
+package teacheasy.data;
+/**
+ * @version 1.0 05 Feb 2015
+ * @author sh1157 and sr896
+ */
+
+/** Class to hold Button Data */
+public class ButtonData extends PageObject {
+	
+	public static enum ButtonInteractionType {
+		NORMAL
+	}
+	
+	float xEnd, yEnd, outlineThickness;
+	boolean outline, shadow, visible;
+	String text;
+	int function;
+	private ButtonInteractionType interaction;

```

Comment:
Lots of these variables are missing private access specifiers.

Style: Generally bad to declare multiple variables on one line if they are not closely related (x and y)